services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/table/TableQueryUtilsTest.java

```diff
@@ -164,4 +169,19 @@ public void testPLFM_6027() {
 		String result = TableQueryUtils.extractTableIdFromSql(sql);
 		assertEquals("syn123", result);
 	}
+
+	@Test
+	/**
+	 * PLFM-6392 Add a more informative message to the user for cases where keywords
+	 * must be escaped.
+	 */
+	public void testUnquotedKeywordsErrorMessage() {
+		String sql = "select year from syn123 where year = 1";
+		IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, ()->{
+			TableQueryUtils.extractTableIdFromSql(sql);
+		});
+		assertNotNull(exception);

```

Comment:
This check is not necessary since the assertThrows always returns the exception (otherwise it would fail the test)